### DISCUSSION TITLE
Dual Monitor Setup OTZ doesn't function on Cintiq

### DISCUSSION THREAD

**Discussion by saltiva** [opentoonz/opentoonz#6768] [General]
**Describe the bug**
OpenToonz opens fine on primary monitor (2k monitor) and functions as it should; however when i move it over to Cintiq (4k) it no longer functions correctly- ie/ when i draw a line, the line appears in a different area.  When I open OTZ on Cintiq the app never launches-  black windows and Not Responding.   When i lower the resolution on the Cintiq to match the other monitor the app still crashes upon launch.  If i move the app from primary to Cintiq after launch with matching res the accuracy of the brush is fixed; however, the app doesn't look crisp.  it has distortions etc.  I am not sure if this is a driver issue but i have updated all drivers current.  I tried reinstalling OTZ but this didn't fix the issue.  

**To Reproduce**
List steps to reproduce the behavior.
1. Launch on Cintiq crash every time.
2. Launch on Primary non-Cintiq no issue until moved to Cintiq every time.

**Expected behavior**
I expect the software to behave normally on a Cintiq in 4k or even matching res regardless.  Is this a known issue?  I searched and have not seen this issue mentioned.
![crash](https://user-images.githubusercontent.com/57774811/126023280-06b008f6-c115-422f-9cb6-e1736be9ac6d.jpg)
![Untitled](https://user-images.githubusercontent.com/57774811/126023298-56770310-d6a2-49e4-a6b9-da0af6439d3c.png)

**Screenshots**
If applicable, add screenshots to help explain your problem. Included

**Device Information:**
 - OS: Windows 10
 - OpenToonz Version: 1.5
 - GPU: 3090
 - GPU Driver Version: Current/Latest Game Version

**Comment by RodneyBaker** [comment]
Try mirroring your screens so they are the same resoluation on both monitors.
That may not be optimal but has resolved the problem for users in the past.

You description suggests you might have done this and if so...
With mirroring and same resolution test having each monitor be the first/primary monitor.
